forch/forchestrator.py

```diff
@@ -99,6 +100,8 @@ def initialize(self):
 
         self._validate_config_files()
 
+        self._forch_metrics = ForchMetrics(8302, self._config.http)
```

Comment:
this is not right -- there should only be one config block for this, and the port should be part of it.  I think we might need to do an audit of the config setup -- The current http config makes no sense for this (it's specifying a file root).  If you want to simplify and not make things worse, find a way to do this without passing self._config.http in.

Reply:
Makes sense. Let me look into it.

Reply:
Changed now.